src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.java

```diff
@@ -18,10 +18,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-import org.rstudio.core.client.ColorUtil;
-import org.rstudio.core.client.CommandWithArg;
-import org.rstudio.core.client.Size;
-import org.rstudio.core.client.StringUtil;
+import org.rstudio.core.client.*;
```

Comment:
We usually don't do wildcard imports (there's an IntelliJ setting you can turn off so it doesn't do this)

Reply:
Thanks, fixed this :)